CONTRIBUTING.md

```diff
@@ -14,18 +17,6 @@ See [SCOPE.md](./SCOPE.md) for some guidelines about what we consider in and out
 
 The near-term and mid-term roadmap for the work planned for the project [maintainers](MAINTAINERS.md) is documented in [ROADMAP.md](ROADMAP.md).
 
-## Community Meetings
-
-Pinniped is better because of our contributors and [maintainers](MAINTAINERS.md). It is because of you that we can bring great
-software to the community. Please join us during our online community meetings,
-occurring every first and third Thursday of the month at 9 AM PT / 12 PM ET.
-Use [this Zoom Link](https://go.pinniped.dev/community/zoom)
-to attend and add any agenda items you wish to discuss
-to [the notes document](https://hackmd.io/rd_kVJhjQfOvfAWzK8A3tQ?view).
-Join our [Google Group](https://groups.google.com/g/project-pinniped) to receive invites to this meeting.
```

Comment:
I would keep a variation of this, like "Join our Google Group for invites to detailed release meetings, updates the project, and related conversations.

Reply:
This would be across all the docs where this was changed.

Reply:
Thanks, updated the original commit on this PR to include this.